pkg/database/db/database.go

```diff
@@ -0,0 +1,34 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package db
```

Comment:
I don't think `db` is expressive enough, what about calling this `interfaces` or `types`?

Reply:
I started calling the package `interface` but I didn't like it because even if it's doable, the word `interface` is reserved, so to import it you need always to get an alias. I guess that `interfaces` solves that problem and after reading about general naming conventions in go, the idea is to use singular names except to prevent name collisions (e.g. `strings`, `types`, `errors`...) so I'll go with `interfaces`.

Reply:
Done!